### ISSUE TITLE
[Feature Request]: Add Influxdata Repositories

### ISSUE THREAD

**Issue by Marc-Berg** [buanet/ioBroker.docker#329]
### Description

Please add the Influxdata Repos to the Docker Image. This would make it possible to add the client components (influxdb-client, influxdb2-cli) via the "PACKAGES" environment variable.

Manually I can achieve this using the commands:

```
wget -q https://repos.influxdata.com/influxdata-archive_compat.key
cat influxdata-archive_compat.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg > /dev/null
echo 'deb [signed-by=/etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg] https://repos.influxdata.com/debian stable main' | sudo tee /etc/apt/sources.list.d/influxdata.list
```

**Comment by buanet** [comment]
As discussed at Discord I would prefer not to include third party repos into the docker image by default. 
But maybe we can handle that database backup topic like avahi or z-wave are handled.

**Comment by buanet** [comment]
The suggested feature is available for testing in v8.1.0-beta.2.

So I'll close this now.

Regards,
André
